src/componets/calculator.js

```diff
@@ -0,0 +1,13 @@
+/*eslint-disable */
+
+import "./calculator.css"; 
+
+const Calculator = ({ symbol, Color }) => {
+  return (
+    <div className="calc-wrapper"style={{ backgroundColor:Color}}>
+      {symbol}
+    </div>
+  );
+};
```

Comment:
- [ ] Good work building the calculator component. It is a project requirement that the Calculator component should have at least 1 child component. This means you need to import another component and use it in the calculator class. 🤣